data/json/items/gun/pebble.json

```diff
@@ -0,0 +1,116 @@
+[
+  {
+    "id": "bullet_crossbow",
+    "type": "GUN",
+    "name": "bullet crossbow",
+    "description": "A modified version of the classic crossbow which utilizes stones as projectiles instead of the traditional quarrel.  Primarily intended for hunting small game, stronger people can reload it much faster.",
+    "weight": 1906,
+    "volume": 5,
+    "price": 20000,
+    "bashing": 9,
+    "material": [ "wood", "iron" ],
+    "symbol": "(",
+    "color": "green",
+    "ammo": "pebble",
+    "skill": "archery",
+    "range": 10,
+    "ranged_damage": 10,
+    "dispersion": 55,
+    "durability": 6,
+    "loudness": 3,
+    "clip_size": 1,
+    "reload": 400,
+    "valid_mod_locations": [ [ "accessories", 4 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
+    "flags": [ "STR_RELOAD", "PRIMITIVE_RANGED_WEAPON" ]
+  },
+  {
+    "id": "sling",
+    "type": "GUN",
+    "name": "sling",
+    "//": "It's little more than a piece of slightly shaped leather",
+    "description": "A leather sling, easy to use and accurate.  It uses pebbles as ammunition.",
+    "weight": 96,
+    "volume": 1,
+    "price": 150,
+    "to_hit": -2,
+    "material": "leather",
+    "symbol": "(",
+    "color": "brown",
+    "ammo": "pebble",
+    "skill": "throw",
+    "range": 8,
+    "ranged_damage": 6,
+    "dispersion": 150,
+    "durability": 6,
+    "clip_size": 1,
+    "flags": [ "RELOAD_AND_SHOOT", "NEVER_JAMS", "PRIMITIVE_RANGED_WEAPON", "BELT_CLIP" ]
+  },
+  {
+    "id": "slingshot",
+    "type": "GUN",
+    "name": "slingshot",
+    "description": "A wooden slingshot, easy to use and accurate.  It uses pebbles as ammunition.",
+    "weight": 254,
+    "volume": 1,
+    "price": 500,
+    "to_hit": -2,
+    "material": "wood",
+    "symbol": "(",
+    "color": "brown",
+    "ammo": "pebble",
+    "skill": "archery",
+    "range": 5,
+    "ranged_damage": 3,
+    "dispersion": 75,
+    "durability": 6,
+    "clip_size": 1,
+    "flags": [ "RELOAD_AND_SHOOT", "NEVER_JAMS", "PRIMITIVE_RANGED_WEAPON", "BELT_CLIP" ]
+  },
+  {
+    "id": "tihar",
+    "type": "GUN",
+    "name": "pneumatic assault rifle",
+    "description": "A multistroke pneumatic rifle handcrafted from scrap.  It is very quiet and deadly.",
+    "weight": 4211,
+    "volume": 11,
+    "price": 240000,
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "light_gray",
+    "ammo": "pebble",
+    "skill": "rifle",
+    "range": 15,
+    "ranged_damage": 15,
+    "dispersion": 75,
+    "durability": 7,
+    "burst": 5,
+    "loudness": 5,
+    "clip_size": 15,
+    "reload": 500,
+    "valid_mod_locations": [ [ "accessories", 4 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "sights", 1 ], [ "stock", 1 ] ],
+    "flags": [ "MODE_BURST", "STR_RELOAD", "RELOAD_ONE" ]
+  },
+  {
+    "id": "wristrocket",
```

Comment:
This should be `copy-from sling` with better `range +2` or something along those lines